.stylelintignore

```diff
@@ -5,3 +5,18 @@
 # Unignore css and scss files too.
 !**/*.css
 !**/*.scss
+
+# CSS copied in from elsewhere
+projects/plugins/crm/sass/semantic-ui
+projects/plugins/crm/css/lib
+projects/plugins/crm/css/semantic-ui
```

Comment:
I'll note that technically the crm/css folder is `.gitignore`d, so Stylelint doesn't look at these at the moment. On the other hand, they are committed to the monorepo and probably should not be `.gitignore`...